product_contract_storable/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* `Coop IT Easy SC <https://coopiteasy.be>`_:
+
+  * hugues de keyzer
```

Comment:
i don’t remember working on this module. 😅 did i?

Reply:
fixed